packages/os/bootstrap-containers-toml

```diff
@@ -5,7 +5,9 @@ std = { version = "v1", helpers = ["if_not_null"] }
 {{#if_not_null settings.bootstrap-containers}}
 {{#each settings.bootstrap-containers}}
 ["{{@key}}"]
+{{#if_not_null this.source}}
 source = "{{{this.source}}}"
+{{/if_not_null}}
 mode = "{{{this.mode}}}"
```

Comment:
PR title should be updated. If we need this change, should we also conditionalize the rendering of `mode`? It's strange to assume that's the only one that will always be defined.